### PR TITLE
[SENTINEL] 1.0 release with GDA support

### PR DESCRIPTION
**Pull request by mmd-afegbua**
See https://github.com/superfluid-finance/superfluid-stories/issues/677

- [x] fix tests (broke due to changes in SuperfluidFrameworkDeployer)
- [x] version set to 1.0
- [x] Readme and example .env are up to date (incl. fixing links broken to SF docs update)
- [x] docker-compose files smoke tested
- [x] scripts provided in the `scripts` directory smoke tested

### CODE REVIEW

**Comment by ngmachado** [comment]
The last version of monorepo changed the deployments scripts. We need to update the sentinels tests before merging the PR.

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
   "dependencies": {
     "@decentral.ee/web3-helpers": "^0.5.3",
     "@slack/webhook": "^6.1.0",
-    "@superfluid-finance/ethereum-contracts": "1.9.1-dev.083b723.0",
-    "@superfluid-finance/metadata": "^1.1.27",
+    "@superfluid-finance/ethereum-contracts": "1.9.0",
+    "@superfluid-finance/metadata": "1.1.27",
```
Comment (d10r) [inline package.json:31]: is there a reason for this change?

Comment (mmd-afegbua) [reply package.json:31]: I am unable to build on local with that dependency - building of docker image fails, as well as running docker compose config.

Comment (d10r) [reply package.json:31]: I mean the change from `^1.1.27` to `1.1.27` for `@superfluid-finance/metadata`

**Review by ngmachado — COMMENTED** [review]
good pretty solid

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ name: Dev CI/CD
 on:
   push:
     branches:
-      - '*'
```
[the remark this answers is not in the document]
Comment (ngmachado) [reply .github/workflows/ci.yml:6]: We want to run the tests in all branches, not just for master PRs.

**Review by ngmachado — APPROVED** [review]

**Pull request merged**